pkg/ruler/frontend_decoder.go

```diff
@@ -17,45 +17,47 @@ const (
 
 type JsonDecoder struct{}
 type Warning []string
+type Info []string
 
-func (j JsonDecoder) Decode(body []byte) (promql.Vector, Warning, error) {
+func (j JsonDecoder) Decode(body []byte) (promql.Vector, Warning, Info, error) {
 	var response struct {
 		Status    string          `json:"status"`
 		Data      json.RawMessage `json:"data"`
 		ErrorType string          `json:"errorType"`
 		Warnings  Warning         `json:"warnings"`
+		Infos     Info            `json:"infos"`
 		Error     string          `json:"error"`
```

Comment:
We probably missed this during the code review. Can we use https://github.com/cortexproject/cortex/blob/master/pkg/util/api/response.go#L20 struct instead?

Reply:
Let me fix it